ddpui/core/notifications_service.py

```diff
@@ -126,11 +118,23 @@ def create_notification(
     if not notification:
         return {"message": "Failed to sent notification."}, None
 
+    last_recipient_id = None
     for recipient_id in recipients:
+        last_recipient_id = recipient_id
         error = handle_recipient(recipient_id, scheduled_time, notification)
```

Comment:
We need to check for all receipeint's org. 
Send the discord notification to all org based on preferences. 
This function doesnt know that all receipients might be from the same org.

Reply:
done